src/decoder_allocators.hpp

```diff
@@ -30,7 +30,7 @@
 #ifndef ZEROMQ_DECODER_ALLOCATORS_HPP
 #define ZEROMQ_DECODER_ALLOCATORS_HPP
```

Comment:
The include guard is off. Pieter: Might be worth codifying the existing practice in the style guide.